ocs-operator-bundle-container fails with runtime error during building

### Environment info
- NooBaa Operator Version: main
- Platform:  OpenShift 
https://jenkins-cpaas-rhodf.apps.cpaas-poc.r6c9.p1.openshiftapps.com/job/rhodf-4.99-rhel-8/job/rhodf/view/Build%20Jobs/job/ocs-operator-bundle-container/job/ocs-operator-bundle-container/5/console

Running 
```
17:16:34  + ./noobaa-operator olm csv
17:16:34  panic: runtime error: invalid memory address or nil pointer dereference
17:16:34  [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x19c2bd6]
17:16:34  
17:16:34  goroutine 1 [running]:
17:16:34  github.com/noobaa/noobaa-operator/v5/pkg/olm.GenerateCSV.func1(0xc0004a1900)
17:16:34  	/remote-source/app/pkg/olm/olm.go:687 +0x3f6
17:16:34  github.com/noobaa/noobaa-operator/v5/pkg/crd.ForEachCRD(0xc000b0f4d0)
17:16:34  	/remote-source/app/pkg/crd/crd.go:166 +0x49
17:16:34  github.com/noobaa/noobaa-operator/v5/pkg/olm.GenerateCSV(0xc0003cd2c0, 0x0)
17:16:34  	/remote-source/app/pkg/olm/olm.go:668 +0x2ab6
17:16:34  github.com/noobaa/noobaa-operator/v5/pkg/olm.RunCSV(0xc00089f180, {0x3299ee0, 0x0, 0x0})
17:16:34  	/remote-source/app/pkg/olm/olm.go:222 +0x25
17:16:34  github.com/spf13/cobra.(*Command).execute(0xc00089f180, {0x3299ee0, 0x0, 0x0})
17:16:34  	/remote-source/deps/gomod/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x5f8
17:16:34  github.com/spf13/cobra.(*Command).ExecuteC(0xc000662780)
17:16:34  	/remote-source/deps/gomod/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x3ad
17:16:34  github.com/spf13/cobra.(*Command).Execute(...)
17:16:34  	/remote-source/deps/gomod/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
17:16:34  github.com/noobaa/noobaa-operator/v5/pkg/cli.Run()
17:16:34  	/remote-source/app/pkg/cli/cli.go:57 +0x1e
17:16:34  main.main()
17:16:34  	/remote-source/app/main.go:13 +0x17
17:16:34  make: *** [Makefile:110: gen-release-csv] Error 2
17:16:34  [Pipeline] }
```

--- a/pkg/olm/olm.go
+++ b/pkg/olm/olm.go
@@ -684,11 +684,11 @@ func GenerateCSV(opConf *operator.Conf, csvParams *generateCSVParams) *operv1.Cl
 		if c.Spec.Group == nbv1.SchemeGroupVersion.Group {
 			csv.Spec.CustomResourceDefinitions.Owned = append(csv.Spec.CustomResourceDefinitions.Owned, crdDesc)
 		} else if c.Spec.Group == obAPI.Domain {
-			if csvParams.OBCMode == OBCOwned {
+			if csvParams != nil && csvParams.OBCMode == OBCOwned {
 				csv.Spec.CustomResourceDefinitions.Owned = append(csv.Spec.CustomResourceDefinitions.Owned, crdDesc)
-			} else if csvParams.OBCMode == OBCRequired {
+			} else if csvParams == nil || csvParams.OBCMode == OBCRequired {
 				csv.Spec.CustomResourceDefinitions.Required = append(csv.Spec.CustomResourceDefinitions.Required, crdDesc)
-			}
+			} // else OBCMode == OBCNone, do nothing
 		} else {
 			csv.Spec.CustomResourceDefinitions.Required = append(csv.Spec.CustomResourceDefinitions.Required, crdDesc)
 		}

